guest/rust/api_core/src/client/experimental/mesh.rs

```diff
@@ -0,0 +1,23 @@
+use crate::global::{Vec2, Vec3};
+use crate::internal::wit;
+
+#[repr(C)]
+#[derive(Clone, Copy)]
+pub struct Vertex {
+    pub position: Vec3,
+    pub normal: Vec3,
+    pub tangent: Vec3,
+    pub texcoord0: Vec2,
+}
+
+pub fn create(vertices: &[Vertex], indices: &[u32]) -> String {
+    let vertices = unsafe {
+        std::slice::from_raw_parts(
```

Comment:
I think we have bytemuck as a dependency? Consider using its `cast_slice` to ensure that size / alignment invariants are kept